wix-events-v2/wix-events-v2/guides/partial-updates.md

```diff
@@ -1,14 +1,16 @@
 # Partial Updates
-Most update endpoints support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
+Most update functions support partial updates via a combination of a _partial entity body_ and a `fields` parameter.
 
 ## Fields parameter
-The `fields` parameter accepts a set of field paths that specifies the entity data to update. In general, the fields path begins from the root of the request body.
+The `fields` parameter accepts a set of field paths that specifies the entity data to update. 
 
-When the `fields` parameter contains a sub-path of the entity fields, like `event` - all nested fields are included in the update: `event.title`, `event.description`, etc.
+The field path begins from the root of the request body and may contain several levels of fields separated by dot notation. This indicates the specific field for update. 
```

Comment:
> root of the request body

the concept of "request body" is a REST term, not Velo.

> separated by dot notation

The fields paths _use_ dot notation or are _in_ dot notation.

> the specific field for update

specific ~~field for~~ **fields to** update